OtripleS.Web.Api.Tests.Unit/Services/ClassroomServiceTests/ClassroomServiceTests.Exceptions.Retrieve.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using OtripleS.Web.Api.Models.Classrooms;
+using OtripleS.Web.Api.Models.Classrooms.Exceptions;
+using Xunit;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.ClassroomServiceTests
+{
+    public partial class ClassroomServiceTests
+    {
+        [Fact]
+        public async Task ShouldThrowDependencyExceptionOnRetrieveByIdWhenSqlExceptionOccursAndLogItAsync()
+        { 
+            // given
+            Guid randomClassroomId = Guid.NewGuid();
```

Comment:
Since this is not critical to the exception being thrown, you can call this:
```csharp
Guid someClassroomId = Guid.NewGuid();
```
And you don't need the `classroomId` because we don't really build any logic on it, it's just a filler-in to cause the exception to occur.

Reply:
i will fix them this evening, thank you.

Reply:
@hassanhabib I have fixed all these issues, please have a look.